### PR TITLE
fix: remove use of theme from logo

### LINKED ISSUE #2307: [refactor] Remove direct usage of Theme from "Logo.tsx"
Remove the following

```ts
// TODO: Remove direct usage of Theme
import { preciousPlasticTheme } from 'oa-themes'
const theme = preciousPlasticTheme.styles
```

At the moment we are importing the theme object directly and accessing its style properties. We should avoid doing this and instead make use of `sx` property available on all ThemeUI components to load these values out of the theme definition.

Read more about it here: 
> The `sx` prop lets you add any valid CSS to an element, while using values from your theme to keep styles consistent. You can think of the style object that the `sx` prop accepts as a _superset_ of CSS.
https://theme-ui.com/sx-prop

The aim should be for there to be no visual differences. However there may be circumstances where this would introduce better visual consistency across the UI. Feel free to reach out on [Discord ](https://discord.com/channels/586676777334865928/938781727017558018) if you think you have found one of those scenarios.

Here's an example where the same change has been made, although I would call out that this specific file required more work than most 🤗  https://github.com/ONEARMY/community-platform/pull/2282

### PR DESCRIPTION
**Pull request by BaltacMihai**
PR Checklist

- [x] - Commit [messages are descriptive](https://github.com/ONEARMY/community-platform/blob/master/CONTRIBUTING.md#--commit-style-guide), it will be used in our [Release Notes](https://github.com/ONEARMY/community-platform/releases/)

PR Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Developer experience (improves developer workflows for contributing to the project)

## Description

Removes direct usage of the theme object, instead accessing it within the sx property.

## Git Issues

Closes #2307

### CODE REVIEW

File: src/pages/common/Header/Menu/Logo/Logo.tsx [hunk 1]
```diff
@@ -44,44 +28,49 @@ export class Logo extends Component<IProps> {
...
 
     return (
-      <>
-        <LogoContainer>
-          <Link to="/">
-            <Flex
-              ml={[0, 4]}
+      <Box
+        sx={{
+          py: ['10px', '10px', 0], // padding on y axes ( top & bottom )
```
Comment (thisislawatts) [inline src/pages/common/Header/Menu/Logo/Logo.tsx:37]: **suggestion:** You can reference [the `space` array available in the theme](https://github.com/ONEARMY/community-platform/blob/f5e1444ef627840a91c4dca14007ffbfa30d3855/packages/themes/src/project-kamp/styles.ts#L32-L35).

```suggestion
          py: [2, 2, 0], // padding on y axes ( top & bottom )
```

File: src/pages/common/Header/Menu/Logo/Logo.tsx [hunk 2]
```diff
@@ -44,44 +28,49 @@ export class Logo extends Component<IProps> {
...
+      >
+        <Link to="/">
+          <Flex
+            ml={[0, 4]}
+            sx={{
+              zIndex: 1000,
+              display: 'flex',
+              alignItems: 'center',
+              width: ['50px', '50px', '100px'],
+              height: ['50px', '50px', '100px'],
```
Comment (thisislawatts) [inline src/pages/common/Header/Menu/Logo/Logo.tsx:50]: **question:** You seem to be including the `px` value here but not when declaring the width on the `<Image/>` element. Is that intentional?

File: src/pages/common/Header/Menu/Logo/Logo.tsx [hunk 3]
```diff
@@ -44,44 +28,49 @@ export class Logo extends Component<IProps> {
...
-                title={nameAndVersion}
-              />
-            </Flex>
-            <Text
-              className="sr-only"
-              ml={2}
-              sx={{ display: ['none', 'none', 'block'] }}
-              color="black"
-            >
-              Precious Plastic
```
Comment (thisislawatts) [inline src/pages/common/Header/Menu/Logo/Logo.tsx:80]: **praise:** Great catch on this @BaltacMihai ✨ Thanks for fixing this!

File: src/pages/common/Header/Menu/Logo/Logo.tsx (hunk 1 above)
Comment (BaltacMihai) [reply src/pages/common/Header/Menu/Logo/Logo.tsx:37]: To be honest, the pixel value makes it simpler to read, but I can adjust

Comment (thisislawatts) [reply src/pages/common/Header/Menu/Logo/Logo.tsx:37]: Understood, I agree that the theme abstraction takes some getting used to, but it means that if we wanted to change our 10px value to 9px we would only need to do so in the Theme configuration and not across all of our components.

Comment (BaltacMihai) [reply src/pages/common/Header/Menu/Logo/Logo.tsx:37]: I get it now.

File: src/pages/common/Header/Menu/Logo/Logo.tsx (hunk 2 above)
Comment (BaltacMihai) [reply src/pages/common/Header/Menu/Logo/Logo.tsx:50]: It wasn't done on purpose. In order to implement the DRY principle (don't repeat yourself), I create one variable everytime the array is repeated.

**Review by thisislawatts — APPROVED** [review]

**Comment by thisislawatts** [comment]
@all-contributors add @BaltacMihai for code

**Pull request merged**